### ISSUE TITLE
remove master/slave references

### ISSUE THREAD

**Issue by trusktr** [RethinkRobotics-opensource/rosnodejs#151]
It will be more respectful for all possible users to remove the `master` / `slave` terminology from the lib.

https://github.com/RethinkRobotics-opensource/rosnodejs/blob/af5d458829fe8103ea45fcd51498140d2b7cba85/src/lib/RosNode.js#L22-L23

PR with initial replacements: https://github.com/RethinkRobotics-opensource/rosnodejs/pull/152. I'm not sure what will break yet.

:black_large_square:

**Comment by chfritz** [comment]
Would it make sense to wait and see how ROS core decides to resolve this issue first? I think it would be very confusing to use different terminology in the ROS libraries than in [ROS core](http://wiki.ros.org/ROS/Master_Slave_APIs).

**Comment by trusktr** [comment]
I have a feeling it will take more time for ROS core, because it may be a huge breaking change. Do you know if there's interest there?

**Comment by chfritz** [comment]
No, I haven't spoken with anyone at OSRF lately. I agree though that is could end up being a huge change. It might even require every ROS user to update their code, since the words in question appear in the API itself.

**Comment by chris-smith** [comment]
I think I'd at least like to hear what ROS thinks. Doing this would break the api and make it harder for new and current users.
